website/docs/en/config/experiments.mdx

```diff
@@ -334,3 +334,150 @@ module.exports = {
   },
 };
 ```
+
+## experiments.cache
```

Comment:
We can follow the alphabetical order and put the `cache` option before `experiments.css`

Reply:
Currently they are arranged in the order of supported versions, we can modify them in another PR

Reply:
ok